scroll-animations-1/Overview.bs

```diff
@@ -442,7 +446,7 @@ spec: cssom-view-1; type: dfn;
 
 	<pre class='propdef'>
 	Name: view-timeline-name
-	Value: none | <<custom-ident>>#
+	Value: none | <<single-timeline-name>>#
```

Comment:
Actually, I think `none` needs to be inside the list, otherwise it's not possible to represent the following as a shorthand: `view-timeline-axis:block,inline; view-timeline-name:none`.

Reply:
OK, disregard that, we can't represent different-length longhands as a shorthand anyway ...